### ISSUE TITLE
[BUG] Example notebooks do not work

### ISSUE THREAD

**Issue by jnehring** [pysentimiento/pysentimiento#62]
**Describe the bug**
I ran the [example notebook](https://colab.research.google.com/github/pysentimiento/pysentimiento/blob/master/notebooks/examples/pysentimiento_sentiment_analysis_in_spanish.ipynb) and it failed in step 2.

**To Reproduce**

```
!pip install pysentimiento
from pysentimiento import create_analyzer
import transformers

transformers.logging.set_verbosity(transformers.logging.ERROR)

analyzer = create_analyzer(task="sentiment", lang="es")
```

Screenshot
<img width="1001" alt="image" src="https://github.com/pysentimiento/pysentimiento/assets/10537540/47065442-3304-4047-b8f8-5be0fda59cc1">

**Comment by finiteautomata** [comment]
It seems that it is an incompatibility with [`accelerate`](https://github.com/huggingface/transformers/issues/29216). I'll update the packages and check if it works :-)

**Comment by jnehring** [comment]
it works for me too, thank you very much :)

**Comment by xiaxin1998** [comment]
> Fixed in `0.7.3`.

what do you mean here? change accelerate to 0.7.3?

**Comment by finiteautomata** [comment]
It means that it is fixed in version `0.7.3` of `pysentimiento`
